dynawo/sources/Models/Modelica/Dynawo/Electrical/HVDC/Standard/HVDCStandard.mo

```diff
@@ -0,0 +1,160 @@
+    Placement(visible = true, transformation(origin = {130, -8}, extent = {{-40, -40}, {40, 40}}, rotation = 0), iconTransformation(origin = {110, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+
+  parameter Types.PerUnit IpMaxcstPu "Maximum value of the active current in p.u (base SNom, UNom)";
+  parameter Types.CurrentModulePu InPu "Nominal current in p.u (base SNom, UNom)";
+  parameter Types.ApparentPowerModule SNom "Nominal apparent power in MVA";
+  extends Parameters.Params_DCLine;
+  extends Parameters.Params_ActivePowerControl;
+  extends Parameters.Params_DCVoltageControl;
+  extends Parameters.Params_BlockingFunction;
+  extends Parameters.Params_ACVoltageControl;
+
+  Modelica.Blocks.Interfaces.RealInput QRef1Pu(start = - Q10Pu / (SNom/SystemBase.SnRef)) "Reactive power reference for the side 1 of the HVDC link in p.u (base SNom) and in generator convention" annotation(
+    Placement(visible = true, transformation(origin = {-50, 77}, extent = {{-7, -7}, {7, 7}}, rotation = -90), iconTransformation(origin = {-90, 110}, extent = {{-10, -10}, {10, 10}}, rotation = -90)));
+  Modelica.Blocks.Interfaces.RealInput PRefPu(start = - P10Pu / (SNom/SystemBase.SnRef)) "Active power reference of the HVDC link in p.u (base SNom) and in generator convention" annotation(
+    Placement(visible = true, transformation(origin = {-60, 77}, extent = {{-7, -7}, {7, 7}}, rotation = -90), iconTransformation(origin = {-50, 110}, extent = {{-10, -10}, {10, 10}}, rotation = -90)));
+  Modelica.Blocks.Interfaces.RealInput URef1Pu(start = U10Pu - Lambda * Q10Pu / (SNom/SystemBase.SnRef)) "Voltage reference for the side 1 of the HVDC link in p.u (base UNom)" annotation(
+    Placement(visible = true, transformation(origin = {-40, 77}, extent = {{-7, -7}, {7, 7}}, rotation = -90), iconTransformation(origin = {-70, 110}, extent = {{-10, -10}, {10, 10}}, rotation = -90)));
+  Modelica.Blocks.Interfaces.RealInput modeU1(start = 1) "Real assessing the mode of the control: 1 if U mode, 0 if Q mode" annotation(
+    Placement(visible = true, transformation(origin = {-30, 77}, extent = {{-7, -7}, {7, 7}}, rotation = -90), iconTransformation(origin = {-30, 110}, extent = {{-10, -10}, {10, 10}}, rotation = -90)));
+  Modelica.Blocks.Interfaces.RealInput QRef2Pu(start = - Q20Pu / (SNom/SystemBase.SnRef)) "Reactive power reference for the side 2 of the HVDC link in p.u (base SNom) and in generator convention" annotation(
+    Placement(visible = true, transformation(origin = {50, 77}, extent = {{-7, -7}, {7, 7}}, rotation = -90), iconTransformation(origin = {30, 110}, extent = {{-10, -10}, {10, 10}}, rotation = -90)));
+  Modelica.Blocks.Interfaces.RealInput URef2Pu(start = U20Pu - Lambda * Q20Pu / (SNom/SystemBase.SnRef)) "Voltage reference for the side 2 of the HVDC link in p.u (base UNom)" annotation(
```

Comment:
It gives the same result but I find it easier to understand to have x0Pu * SnRef / SNom. Otherwise, I always have difficulties going through the x / (y / z) structure. (In addition if it remains divisions, divisions are more costly in cpu times than multiplication ;))

Reply:
Done